apps/src/templates/courseOverview/CourseOverviewTopRow.js

```diff
@@ -47,6 +49,9 @@ export default class CourseOverviewTopRow extends Component {
             color={Button.ButtonColor.blue}
           />
         ))}
+        {experiments.isEnabled(experiments.ASSIGNMENT_UPDATES) && (
+          <SectionAssigner sections={sectionsInfo} />
+        )}
```

Comment:
I think it would be best to also conditionally hide the old assignment button when this experiment is enabled. that way, when we launch, it's easy to verify the post-launch behavior, and we also won't have to remember to go and remove that button at that time.

Reply:
Good call. I'll hide the old assign dropdown button and the old section selector so we can get a better feel for the new UI.

Reply:
updated and included a new screenshot.